x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -89,27 +87,15 @@ const Filters = ({ filters, dataViewId, index, 'data-test-subj': dataTestSubj }:
   const flattenedFilters = mapAndFlattenFilters(filters);
 
   return (
-    <EuiFlexGroup wrap responsive={false} gutterSize="xs" data-test-subj={dataTestSubj}>
-      {flattenedFilters.map((filter, idx) => {
-        const displayContent = filter.meta.alias ? (
-          filter.meta.alias
-        ) : (
-          <FilterBadgeGroup filters={[filter]} dataViews={[indexPattern]} />
-        );
-        return (
-          <EuiFlexItem
-            grow={false}
-            key={`filter-${idx}`}
-            css={{ width: '100%' }}
-            data-test-subj={`filterItem-${filter.meta.key}`}
-          >
-            <EuiBadgeWrap color="hollow">
-              {indexPattern != null ? displayContent : <EuiLoadingSpinner size="m" />}
-            </EuiBadgeWrap>
-          </EuiFlexItem>
-        );
-      })}
-    </EuiFlexGroup>
+    <FiltersWidthLimiter>
+      <EuiFlexGroup wrap={true} responsive={false} gutterSize="xs" data-test-subj={dataTestSubj}>
```

Comment:
nit: `={true}` is optional for boolean properties. `wrap={true}` and `readOnly={true}` can be replaced with just `wrap` and `readOnly`.